### PR TITLE
solved issue #388 adding the restartPolicy and removing the write permissions check for the packs volumes

### PR DESCRIPTION
**Pull request by cayohollanda**

### CODE REVIEW

**Comment by rafa13bb** [comment]
Resolved the issue locally in my tests

**Review by cognifloyd — CHANGES_REQUESTED** [review]
I'm not ok with removing the error message and instructions on how to resolve issues with this.

Do you still see these errors now that #403 has been merged? What error messages are you seeing?

File: templates/jobs.yaml [hunk 1]
```diff
@@ -573,6 +552,7 @@ spec:
           readOnly: false
         # TODO: Find out default resource limits for this specific job (#5)
         #resources:
+      restartPolicy: OnFailure
```
Comment (cognifloyd) [inline templates/jobs.yaml:555]:
```suggestion
```

This was added in #403. You can see it below on line 584 below.

https://github.com/StackStorm/stackstorm-k8s/blob/fabbea9fb89bce6d093ad718def3e28fcfc47c5e/templates/jobs.yaml#L582-L586

Comment (fdrab) [reply templates/jobs.yaml:555]: one more thing required for this to not fail is to remove the 2 leading whitespaces before FAIL on line 557. unless someone does it before me, I'll submit a PR once my company approves me signing of the CLA

Comment (cognifloyd) [reply templates/jobs.yaml:555]: Cool. I have a fix for that. I'll push a PR in a few minutes.

Comment (cognifloyd) [reply templates/jobs.yaml:555]: Fix pushed in #411

**Pull request closed without merging**